### PR TITLE
Update install instructions

### PR DESCRIPTION
**Pull request by roigcarlo**
**📝 Description**
Moved MinGW to a separate category.
Changes GNU/Linux to Linux/WSL, which is more useful for the users.

@loumalouomega I know that is a subsection of windows, but we are having lots of users which are mixing MSVC with MINGW, so I prefer to split it clearly

### CODE REVIEW

File: INSTALL.md [hunk 1]
```diff
@@ -6,11 +6,11 @@
     - [Specific Application Dependencies](#specific-application-dependencies)
   - [Basic Configuration](#basic-configuration)
   - [Configuration scripts examples](#configuration-scripts-examples)
-    - [GNU/Linux](#gnulinux)
+    - [Linux/WSL](#gnulinux)
```
Comment (loumalouomega) [inline INSTALL.md:9]: But its GNU/Linux, so I need to mention Stallman?

**Review by loumalouomega — APPROVED** [review]
As long as it doesn't remove it...

**Comment by roigcarlo** [comment]
To be honest I've never seen the real use of mingw builds, even less now that we have WSL... but I have no problem leaving it as long as it is clearly separated from the Win section

**Comment by loumalouomega** [comment]
> To be honest I've never seen the real use of mingw builds, even less now that we have WSL... but I have no problem leaving it as long as it is clearly separated from the Win section

It is far more performant than WSL..., and from scratch is easier to configure than VS, take far less space and doesn't require admin rights.

**Comment by roigcarlo** [comment]
Well ease to use is arguable, and I don't have performance metrics of WSL/WSL2/CYGWIN/MINGW to compare (It is a an interesting metric, but so far I have seen no results comparing this).

The only points I could se with mingw is:
- To generate portable native builds of Kratos under windows. Anyone interested in those probably uses linux.
- To generate binaries that depend on windows components (WIN32's GUI mostly). Not for kratos.

**Pull request merged**